shell/packages/sandstorm-permissions/permissions-tests.js

```diff
@@ -0,0 +1,137 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var globalDb = new SandstormDb();
+
+globalDb.collections.grains.remove({});
```

Comment:
If it's a fresh isolated mongo then why do you need to clear it? :)

I mean that each test case ought to create its own DB instance which probably should not actually be backed by mongo, but instead be a fake in-memory thing that behaves like a real db. I guess in testing terminology the right word is "fake" rather than "mock" -- a "mock" object is one where your test actually tells it exactly what sequence of method calls to expect, whereas a "fake" is a lightweight object that behaves similarly to the "real thing" but for testing purposes.